unison-src/transcripts/constructor-applied-to-unit.output.md

```diff
@@ -26,27 +26,29 @@ structural type Zoink a b c = Zoink a b c
           ⧩
           [ Zoink
               [0, 1, 2, 3, 4, 5]
-              [ 6,
-                3,
-                3,
-                3,
-                3,
-                3,
-                3,
-                3,
-                3,
-                3,
-                3,
-                4,
-                4,
-                4,
-                4,
-                4,
-                4,
-                4,
-                4,
-                4,
-                3 ]
-              () ]
+              [ 6
+              , 3
+              , 3
+              , 3
+              , 3
+              , 3
+              , 3
+              , 3
+              , 3
+              , 3
+              , 3
+              , 4
+              , 4
+              , 4
+              , 4
+              , 4
+              , 4
+              , 4
+              , 4
+              , 4
+              , 3
+              ]
```

Comment:
New rendering of lists (and tuples get same treatment). Easier to find the extent of the list - the `,` acts as a margin. Closing brace goes on its own line.